Makefile

```diff
@@ -57,6 +61,13 @@ clang-tidy:
 	# DeprecatedOrUnsafeBufferHandling: See https://stackoverflow.com/a/50724865/945780
 	clang-tidy psi-notify.c -checks=-clang-analyzer-security.insecureAPI.DeprecatedOrUnsafeBufferHandling -- $(CFLAGS) $(LDFLAGS)
 
+install: all
+	mkdir -p $(DESTDIR)$(bindir)/
+	$(INSTALL) -pt $(DESTDIR)$(bindir)/ $(EXECUTABLES)
+ifeq ($(HAS_LIBSYSTEMD),1)
```

Comment:
It's conceivable that someone might not want libsystemd integration but still wants the service file. So maybe just install it unconditionally? :-)